### ISSUE TITLE
Error building `sprs` 

### ISSUE THREAD

**Issue by MashyBasker** [vbarrielle/sprs#344]
### What I did

- Forked and cloned the project
- run `cargo build` inside the project

### Error encountered

- Could not find the file `Eigen/SparseCore`

![image](https://github.com/sparsemat/sprs/assets/84385565/ab5c9d81-0939-418e-a57e-ae30ffc65c4b)

### What I tried

- Try to compile it with Rust nightly(did not work)
- Tried `export dl_eigen=1`(did not work)
- Install `eigen2` from AUR(did not work)

**Comment by MashyBasker** [comment]
After running the two commands you provided, the build process was successful. However, when using `cargo build --workspace --features sprs-benches`, additional packages were installed, notably `vcpkg`. Consequently, I assumed that `eigen` would also be installed. Nevertheless, upon running just cargo build again, the same error as originally posted occurred.

**Comment by mulimoen** [comment]
Packages are not installed system-wide, you will need to specify `--features sprs-benches/dl_eigen` for every `cargo` invocation, unless you install `eigen` yourself (package name is [`eigen`](https://archlinux.org/packages/extra/any/eigen/)).

**Comment by MashyBasker** [comment]
Yes. I've made a [PR](https://github.com/sparsemat/sprs/pull/345) for future users who encounter this error while building. Please inform me if I need to make some edits.
